src/app/components/popup-choose-the-time/orderby.pipe.ts

```diff
@@ -0,0 +1,13 @@
+// import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
Delete it or uncomment if it uses somewhere

Reply:
Thank you for your comment. I have corrected.